ocean_lib/ocean/test/test_ocean_assets.py

```diff
@@ -254,13 +254,36 @@ def test_create_asset_with_owner_address(publisher_ocean_instance):
     my_secret_store = "http://myownsecretstore.com"
     auth_service = ServiceDescriptor.authorization_service_descriptor(my_secret_store)
 
+    token = ocn.create_data_token(
+        "DataToken1", "DT1", from_wallet=alice, blob="foo_blob"
+    )
+
     assert ocn.assets.create(
         asset.metadata,
         alice,
         [auth_service],
         owner_address=alice.address,
     )
 
+    asset_1 = ocn.assets.create(
+        asset.metadata,
+        alice,
+        [auth_service],
+        owner_address=alice.address,
+        data_token_address=token.address,
+    )
+    assert asset_1
+    asset_2 = ocn.assets.create(
+        asset.metadata,
+        alice,
+        [auth_service],
+        data_token_address=token.address,
+    )
+    assert asset_2
+    assert (
+        asset_1.__dict__["_proof"]["creator"] == asset_2.__dict__["_proof"]["creator"]
```

Comment:
can you use the object instead of the __dict__ representation? Using __dict__ is more like a shortcut for testing, when you don't know the structure of the object closely. But it's better to check with object properties.